docs/examples/floating-pv/plot_floating_pv_cell_temperature.py

```diff
@@ -0,0 +1,217 @@
+r"""
+Calculating the cell temperature for floating PV
+================================================
+
+This example demonstrates how to calculate the cell temperature for
+floating photovoltaic (FPV) systems using the PVSyst temperature model.
```

Comment:
```suggestion
This example uses the PVSyst temperature model to calculate the
cell temperature for floating photovoltaic (FPV) systems.
```

This is just of my personal taste. I feel like fitting example descriptions to the preview cards is underrated. I am part of the problem thou.

![image](https://github.com/pvlib/pvlib-python/assets/80125792/7ae54ffb-5c81-40ad-bf53-a885e8fc7268)